media/js/base/mozilla-monitor-button.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+Mozilla.MonitorButton = (function() {
+    'use strict';
+
+    var monitorButton = document.getElementById('fxa-monitor-submit');
+    var buttonURL = monitorButton.getAttribute('href');
+
+    // fetch request
+    var supportsFetch = 'fetch' in window;
+
+    if (!supportsFetch) {
+        return;
+    }
+
+    var destURL = monitorButton.getAttribute('action') + 'metrics-flow';
```

Comment:
I don't think action is a valid attribute for `a` and it does get used on `form` so I'm worried about confusing the browser. Maybe you want `data-action` instead?